/**:
  ros__parameters:
    left_target_pose_list:
    #   x, y, z, roll, pitch, yaw, grasp
      - "0.5, -0.5, 0.2, 0, 3.14, 0.0" # 2
      - "0.5, -0.45, 0.2, 0, 3.14, 0.0" # 1
      - "0.5, -0.475, 0.28, 0, 3.14, 0.0" # 4
    right_target_pose_list:
    #  x, y, z, roll, pitch, yaw, grasp
      - "0.5, 0.45, 0.2, 0, 3.14, 0.0" # 3
      - "0.5, 0.475, 0.28, 0, 3.14, 0.0" # 5
      - "0.5, 0.5, 0.35, 0, 3.14, 0.0" # 6

#     6
#   4   5
# 1   2   3